crates/matrix-sdk-base/Changelog.md

```diff
@@ -10,7 +10,8 @@
   membership state.
   - `StateStore::get_joined_user_ids` and `StateStore::get_invited_user_ids` are deprecated.
 - `Room::members` takes a `RoomMemberships` to be able to filter the results by any membership
-  state. 
+  state.
+  - `Room::active_members` and `Room::joined_members` are deprecated.
```

Comment:
I'm not sure about this deprecation. It makes sense for internal APIs to be a bit more verbose so they can be more expressive, but for public API it might be nice to keep the shorter ones? How about updating the docs to say they're just shorthands instead?

Reply:
Sure, it's just something you seemed to agree on in #930. To me the general consensus is that there are already too many functions in the API, and we need to add more for other features so reducing a feature to a single method solves that.

Reply:
Alright, let's keep it as-is I guess.